src/handlers/prioritize.rs

```diff
@@ -35,9 +35,15 @@ impl Handler for PrioritizeHandler {
         if let Event::Issue(e) = event {
             if e.action == github::IssuesAction::Labeled {
                 if let Some(config) = config {
-                    if e.label.as_ref().expect("label").name == config.label {
-                        // We need to take the exact same action in this case.
-                        return Ok(Some(Prioritize::Start));
+                    if let Ok(glob) = glob::Pattern::new(&config.priorities_pattern) {
+                        let label_name = &e.label.as_ref().expect("label").name;
+
+                        if glob.matches(label_name)
+                            && config.prioritize_on.iter().any(|l| l == label_name)
+                        {
+                            // We need to take the exact same action in this case.
+                            return Ok(Some(Prioritize::Start));
+                        }
```

Comment:
This doesn't seem correct to me. We need to keep the old behaviour, i.e. if the label name is the same as the label in config, start the prioritization. What I think we want to do is, if it doesn't match the prioritization label, check if it matches any of the `prioritize_on` labels. Then if it does, check the other labels (already on the issue) and match them against the pattern defined in `config.priorities_pattern`. If no labels match, add the label by returning `Prioritize::Label` (and not `Prioritize::Start`).

PS: just to clarify, `Prioritize::Start` sends the message on Zulip, whereas `Prioritize::Label` adds the labels on the GH issue (thus triggers `Prioritize::Start` later).

Reply:
Fixed too.